packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,646 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from 'react';
+import cx from 'classnames';
+import {
+  useSafeContext,
+  // useTheme,
+  SvgMore,
+  type PolymorphicComponentProps,
+  type PolymorphicForwardRefComponent,
+  StatusIconMap,
+  SvgCheckmark,
+  SvgNew,
+  LinkAction,
+  supportsHas,
+  polymorphic,
+} from '../utils/index.js';
+import '@itwin/itwinui-css/css/tile.css';
+import { ProgressRadial } from '../ProgressIndicators/index.js';
+import { DropdownMenu } from '../DropdownMenu/index.js';
+import { IconButton } from '../Buttons/index.js';
+// import { ProgressRadial } from '../ProgressIndicators';
+// import Tile from '.';
+
+const TileContext = React.createContext<
+  | {
+      status: 'positive' | 'warning' | 'negative' | undefined;
+      /**
+       * Whether the tile is selected or in "active" state.
+       * Gets highlighted and shows a checkmark icon near tile name.
+       */
+      isSelected?: boolean;
+      /**
+       * Whether the tile is "new". Tile name becomes bold and gets a new status icon.
+       */
+      isNew?: boolean;
+      /**
+       * Default tile variant or the folder layout.
+       * @default 'default'
+       */
+      variant?: 'default' | 'folder';
+      /**
+       * Whether the tile is expected to be interactable (i.e. `onClick`).
```

Comment:
interactable wasn't coming up as a word in the spellchecker. I think it can be changed to `interactive`. There's another instance in this file where this change can be made as well

Reply:
Thanks for catching. I updated both spots.